plugins/cost-insights/src/types/index.ts

```diff
@@ -14,22 +14,40 @@
  * limitations under the License.
  */
 
+import * as common from '@backstage/plugin-cost-insights-common';
+/** * @deprecated use the same type from @backstage/plugin-cost-insights-common instead */
```

Comment:
extra asterisk at the start

also, i see new warnings in the api-docs too, unfortunately you seem to need to wrap the package name in backticks, so ``` `@backstage/plugin-cost-insights-common` ```

Reply:
Fixed in #12396